test-cypress/integration/explore/class.hierarchy.spec.js

```diff
@@ -1,180 +1,179 @@
+
+        // Verify that prefixes are removed from diagram
+        verifyPrefixes(($element) => expect($element.text()).to.not.contain(':'));
     });
 
-    it('Test reload diagram', function () {
+    it('Test focus on diagram', () => {
         let className = ':CabernetSauvignon';
+        findClassByName(className);
+        cy.get('@classInHierarchy').then(verifyClassIsNotExpanded);
 
-        //  Confirm the initial state of the diagram has a class count of 50
-        cy.get('.rz-pointer[role="slider"]')
-            .then(($element) => {
-                expect($element.prop('aria-valuenow') === 50);
-            });
+        // Verify that the diagram zooms that class
+        searchForClass(className);
 
-        // Change the class count to custom value
-        cy.get('.class-cnt-slider > .ng-isolate-scope')
-            .trigger('click', 'center', {force: true});
+        // Verify font-size is changed
+        findClassByName(className);
+        cy.get('@classInHierarchy').then(verifyClassIsExpanded);
 
-        // Select a class ":CabernetSauvignon"
-        cy.get('#main-group > text.label')
-            .each(($element) => {
-                let data = $element.prop('__data__');
-                if (data.name === className) {
-                    let fontSize = $element.prop('style');
-                    expect(fontSize['font-size']).be.eq('10px');
-                }
-            });
-        // Verify that the diagram zooms that class
-        cy.get('.p-0 > .icon-search')
-            .click()
-            .then(() => {
-                cy.get('#search_input_value').type(className).type('{enter}').wait(500);
-            });
+        // When a class is focused in diagram a side panel is opened on the right and covers the
+        // buttons toolbar.
+        cy.get('[ps-class=rdf-info-side-panel] .close').click()
+            .should('not.be.visible');
+        cy.get('.toolbar-holder .focus-diagram-btn').click().wait(500);
 
-        // Find class ":CabernetSauvignon" and verify changed font-size
-        cy.get('#main-group > text.label')
-            .each(($element) => {
-                let data = $element.prop('__data__');
-                if (data.name === className) {
-                    let fontSize = $element.prop('style');
-                    expect(fontSize['font-size']).not.be.eq('10px');
-                }
-            });
+        // Verify that the diagram zooms out without resetting the class count.
+        findClassByName(className);
+        cy.get('@classInHierarchy').then(verifyClassIsNotExpanded);
+    });
 
-        // Click on "Reload diagram"
-        cy.get('#toolbar > [tooltip="Reload diagram"] > .icon-reload').click({force: true}).wait(200);
+    it('Test reload diagram', () => {
+        // Initial class count is 50
+        // Change the class count to a custom value
+        cy.get('.class-cnt-slider > .ng-isolate-scope')
+            .trigger('click', 'center', {force: true});
```

Comment:
I like to document why force is needed.

Reply:
Honestly I don't know. I decided to leave this action as is because it operates on a third party library realizing the slider and I didn't want to waste time to see why its needed initially.

Reply:
I removed it and it didn't fail. Will see.